### ISSUE TITLE
Error when build application on OS X 10.11 El Capitan

### ISSUE THREAD

**Issue by RomanGorbatko** [transistorsoft/cordova-background-geolocation-lt#14]
```
$ ionic build ios
...
background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSReachability.o)
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSLocationManager.o)
duplicate symbol _llvm.embedded.module in:
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSReachability.o)
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSLocationManager.o)
duplicate symbol _llvm.cmdline in:
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSReachability.o)
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(LocationDAO.o)
duplicate symbol _llvm.embedded.module in:
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSReachability.o)
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(LocationDAO.o)
duplicate symbol _llvm.cmdline in:
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSReachability.o)
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(SOMotionDetector.o)
duplicate symbol _llvm.embedded.module in:
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSReachability.o)
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(SOMotionDetector.o)
ld: 12 duplicate symbols for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)

** BUILD FAILED **


The following build commands failed:
	Ld build/emulator/background.app/background normal i386
(1 failure)
Error code 65 for command: xcodebuild with args: -xcconfig,/Users/rg/Devs/Apps/background/platforms/ios/cordova/build-debug.xcconfig,-project,background.xcodeproj,ARCHS=i386,-target,background,-configuration,Debug,-sdk,iphonesimulator,build,VALID_ARCHS=i386,CONFIGURATION_BUILD_DIR=/Users/rg/Devs/Apps/background/platforms/ios/build/emulator,SHARED_PRECOMPS_DIR=/Users/rg/Devs/Apps/background/platforms/ios/build/sharedpch
ERROR building one of the platforms: Error: /Users/rg/Devs/Apps/background/platforms/ios/cordova/build: Command failed with exit code 2
You may not have the required environment or OS to build this project
Error: /Users/rg/Devs/Apps/background/platforms/ios/cordova/build: Command failed with exit code 2
    at ChildProcess.whenDone (/usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/cordova/superspawn.js:139:23)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at maybeClose (internal/child_process.js:817:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)

```

```
$ ionic info

Your system information:

Cordova CLI: 5.3.3
Gulp version:  CLI version 3.9.0
Gulp local:   Local version 3.9.0
Ionic Version: 1.1.0
Ionic CLI Version: 1.6.5
Ionic App Lib Version: 0.3.9
ios-deploy version: 1.7.0 
ios-sim version: 5.0.1 
OS: Mac OS X Yosemite (or El Capitan)
Node Version: v4.1.1
Xcode version: Xcode 6.4 Build version 6E35b
```

**Comment by christocracy** [comment]
https://github.com/transistorsoft/cordova-background-geolocation-lt/wiki/Help-it-Doesn't-Work!!#when-in-doubt-remove--re-install-the-plugin-and-platform

**Comment by christocracy** [comment]
Specifically, the "When in Doubt" bit.

**Comment by RomanGorbatko** [comment]
Don't work for me.

**Comment by christocracy** [comment]
Does it do same in my SampleApp?

https://github.com/transistorsoft/cordova-background-geolocation-SampleApp

On Tuesday, October 6, 2015, Roman Gorbatko <notifications@github.com>
wrote:

> Don't work for me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/transistorsoft/cordova-background-geolocation-lt/issues/14#issuecomment-146018983>
> .
>


-- 
Snet form Gmail Mobile

**Comment by RomanGorbatko** [comment]
Yes, I have the same error. :(

**Comment by christocracy** [comment]
You need to upgrade Xcode to 7.

**Comment by RomanGorbatko** [comment]
> You need to upgrade Xcode to 7
Fine! It's works.

**Comment by nschurmann** [comment]
same here!, with xcode 8.2.1

**Comment by christocracy** [comment]
@nschurmann you have the *identical* build error?

**Comment by nschurmann** [comment]
The error:

** BUILD FAILED **


The following build commands failed:
	Ld build/emulator/background.app/background normal x86_64
(1 failure)

Error: Error code 65 for command: xcodebuild with args: -xcconfig,/Users/nicolas/background/background/platforms/ios/cordova/build-debug.xcconfig,-workspace,background.xcworkspace,-scheme,background,-configuration,Debug,-sdk,iphonesimulator,-destination,platform=iOS Simulator,name=iPhone 5s,build,CONFIGURATION_BUILD_DIR=/Users/nicolas/background/background/platforms/ios/build/emulator,SHARED_PRECOMPS_DIR=/Users/nicolas/background/background/platforms/ios/build/sharedpch

**Comment by nschurmann** [comment]
While trying to build it for iOS.

**Comment by christocracy** [comment]
This issue is very old, dating back to Oct 2015.

Are you seeing this?
```
background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSReachability.o)
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSLocationManager.o)
duplicate symbol _llvm.embedded.module in:
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSReachability.o)
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSLocationManager.o)
duplicate symbol _llvm.cmdline in:
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSReachability.o)
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(LocationDAO.o)
duplicate symbol _llvm.embedded.module in:
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSReachability.o)
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(LocationDAO.o)
duplicate symbol _llvm.cmdline in:
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSReachability.o)
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(SOMotionDetector.o)
duplicate symbol _llvm.embedded.module in:
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(TSReachability.o)
    background/Plugins/com.transistorsoft.cordova.background-geolocation/TSLocationManager.framework/TSLocationManager(SOMotionDetector.o)
ld: 12 duplicate symbols for architecture i386
```

**Comment by nschurmann** [comment]
Yes!, duplicate symbol error.

**Comment by christocracy** [comment]
Let's create a new issue.  There's an issue template now.  Fill out the template to give me some background on your env.

**Comment by christocracy** [comment]
And btw, I *just* released version `2.4.0` minutes ago with some significant [configuration changes](https://github.com/transistorsoft/cordova-background-geolocation-lt#configuring-the-plugin) as well as a plugin ID change from `com.transistorsoft.cordova.background-geolocation` -> `cordova-background-geolocation`.

We'll continue in the new issue you create...
